### PR TITLE
Async await swap

### PR DESCRIPTION
**Pull request by Georgehe4**
Order of async and await summary notes are swapped.

### CODE REVIEW

**Comment by Georgehe4** [comment]
#484 @devoncarew @lukechurch

File: lib/src/summarize.dart [hunk 1]
```diff
@@ -26,8 +26,8 @@ class Summarizer {
   };
 
   static Map<String, String> codeKeyWords = {
-    'await': 'await',
     'async': 'async',
+    'await': 'await',
```
Comment (devoncarew) [inline lib/src/summarize.dart]: will this change the order in the output?

**Comment by devoncarew** [comment]
lgtm assuming the answer to the above question is yes :)

fix #484

**Comment by Georgehe4** [comment]
Merging.

**Pull request merged**